l10n_th_promptpay/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Poonlap V.
+# Licensed AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Thai Localization - PromptPay",
+    "version": "13.0.1.0.0",
+    "author": "Poonlap V.,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/l10n-thailand",
+    "license": "AGPL-3",
+    "category": "Accounting / Payment",
+    "summary": "Use PromptPay QR code with transfer acquirer.",
+    "depends": ["payment", "payment_transfer", "website_sale"],
```

Comment:
```suggestion
    "depends": ["payment_transfer", "website_sale"],
```
not necessary to add depends by `payment` because `payment_transfer` depends it already.